install.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+# VARIABLES, CHANGE AS NEEDED
+GITHUB_REPO="https://github.com/bmFtZQ/edge-frfox.git"
+PROJECT_NAME=$(basename $GITHUB_REPO | cut -d '.' -f 1)
+
+if [[ $OSTYPE == "darwin"* ]]; then
+  FIREFOX_DIR="$HOME/Library/Application Support/Firefox"
+else
+  FIREFOX_DIR=$HOME/.mozilla/firefox
+fi;
+
+PROFILE_ROOTDIR=$FIREFOX_DIR/$(grep -E "Path=.*\.(dev-edition-default|default-.*)" "$FIREFOX_DIR/profiles.ini" | tail -1 | cut -c 6-);
+OPTIONALS=(
+  "widget.macos.native-context-menus" "false"
+  "browser.theme.dark-private-windows" "false"
```

Comment:
Same as in the powershell script, but with `widget.gtk.rounded-bottom-corners.enabled` = `true` instead.